lib/Api256Shim.ts

```diff
@@ -127,34 +133,62 @@ export class Api256 extends Recrypt.Api256 {
      * Generate an ed25519 signing key pair.
      */
     generateEd25519KeyPair() {
-        const {privateKey, publicKey} = super.generateEd25519KeyPair();
+        const {privateKey, publicKey} = this.api.generateEd25519KeyPair();
         return {
             privateKey: new Uint8Array(privateKey),
             publicKey: new Uint8Array(publicKey),
         };
     }
 
+    /**
```

Comment:
Are the following 4 functions new functionality for the binding?

Reply:
They are no. Previously this class just extended the underlying WASM-generated class (change on line 61 above) and these methods just called directly through to the auto-generated code. But because of some changes to the auto-generated code from `wasm-bindgen` extending the class is no longer possible. So instead I had to write passthrough wrappers for these functions.

Reply:
Makes sense. Thanks for the explanation!